### PR TITLE
Storage pojo and Rest backed

### PR DESCRIPTION
**Pull request by phmp**
Storage class added, REST interface works. Review please.

### CODE REVIEW

**Comment by phmp** [comment]
Adam has added sending rests from webpage. All is on one review.
Adam dodał wysyłanie restów że stronki. Wszystko jest na jednym review.

File: src/main/webapp/resources/js/addOrder.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	var place = listItems[2].value;
+	var description = listItems[3].value;
+	var contact = listItems[4].value;
+
+	var http = new XMLHttpRequest();
+	var url = "/offers";
+	var params = '{"name":"' + name + '","place":"' + place + '","date":"' + date + '","contact":"' + contact + '","description":"' + description + '"}'
+	http.open("POST", url, true);
+	http.setRequestHeader("Content-type", "application/json");
+	/*
```
Comment (phmp) [inline src/main/webapp/resources/js/addOrder.js:17]: commented code should be deleted

File: src/main/webapp/resources/js/addOrder.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+function activateItem() {
+	var name = listItems[0].value;
+	var date = listItems[1].value;
+	var place = listItems[2].value;
+	var description = listItems[3].value;
+	var contact = listItems[4].value;
+
+	var http = new XMLHttpRequest();
+	var url = "/offers";
+	var params = '{"name":"' + name + '","place":"' + place + '","date":"' + date + '","contact":"' + contact + '","description":"' + description + '"}'
```
Comment (phmp) [inline src/main/webapp/resources/js/addOrder.js:14]: Did you run auto-formatter?

**Review by phmp — COMMENTED** [review]
@LordVader14 please check my comments

File: src/main/java/main/RestManager.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.logging.Logger;
+
+import static java.util.logging.Level.*;
+
+/**
+ * Created by Paweł on 2016-12-03.
+ */
+@RestController
+public class RestManager {
```
Comment (LordVader14) [inline src/main/java/main/RestManager.java:20]: Split this class. Create separated container class.

Comment (phmp) [reply src/main/java/main/RestManager.java:20]: ok, done

**Review by gaabral — COMMENTED** [review]
A basic Javadoc for classes (not methods as they are self explanatory) would be useful. I think we should mention who is the author of code - but lets use the same name every time: in this branch there is "epawpro" and "Paweł" used but they mean the same person.

**Review by gaabral — APPROVED** [review]
Add Javadocs and unify author name with next changes.

**Pull request merged**